project/Dependencies.scala

```diff
@@ -97,4 +97,9 @@ object Dependencies {
     "com.google.cloud" % "google-cloud-nio" % Versions.GCNio % IntegrationTest,
     "commons-io" % "commons-io" % "2.6" % Test
   ) ++ testDependencies(hasIt = true)
+
+  val ES = Seq(
+    "com.sksamuel.elastic4s" %% "elastic4s-core" % "6.7.8",
```

Comment:
use latest version? `7.9.1`

Reply:
Version `7.x.x` is incompatibility with version `6.x.x` and older versions. I have no idea to support different ES versions. Any advice?

Reply:
well, a different submodule for each version might be a solution but I prefer to start with a single one and that would rather be the latest. what do you think?

Reply:
OK